src/AlignLoads.cpp

```diff
@@ -0,0 +1,210 @@
+#include "AlignLoads.h"
+#include "IRMutator.h"
+#include "IROperator.h"
+#include "Scope.h"
+#include "Bounds.h"
+#include "ModulusRemainder.h"
+#include "Simplify.h"
+namespace Halide {
+namespace Internal {
+using std::vector;
+
+class AlignLoads : public IRMutator {
+public:
+    AlignLoads(Target t, int vs) : target(t), required_alignment(vs) {}
+private:
+    Target target;
```

Comment:
It looks like target is unused now after removing the For visitor (which is good, but we should get rid of the Target member).